retain-spec/openapi.json

```diff
@@ -9,19 +9,29 @@
     {
       "name": "Account Placements",
       "x-displayName": "Account Placements",
-      "description": "Add and retract accounts."
+      "description": "Place new accounts"
+    },
+    {
+      "name": "Account Updates",
+      "x-displayName": "Account Updates",
+      "description": "Update previously placed accounts"
+    },
+    {
+      "name": "Account Retractions",
+      "x-displayName": "Account Retractions",
+      "description": "Retract previously placed accounts"
     },
     {
       "name": "Payments",
       "x-displayName": "Payments",
```

Comment:
nit: The tuples above are "Account Updates", "Account Retractions" should it be "Account Payments"?